libff/algebra/scalar_multiplication/tests/test_multiexp.cpp

```diff
@@ -388,6 +389,7 @@ int main(int argc, char **argv)
 {
     libff::alt_bn128_pp::init_public_params();
     libff::bls12_377_pp::init_public_params();
+    libff::bls12_381_pp::init_public_params(); // VV
```

Comment:
I think it's my fault, but while this test seems to init multiple curves, it appears that everything is commented except for alt-bn128.  Would you mind uncommenting these and adding the bls12-381 versions as well.  Although the code is basically all generic, I think it's worth running them against bls12-381.

I may have commented these to stop the test taking a crazy amount of time.  Also, some of these may fail due to the signed digits issue.  If that's the case, feel free to just add the bls12-381 tests and keep them commented.  I'll make a note to enable these as part of #71.

Reply:
Actually, @vesselinux , feel free to ignore this - it's probably not fair to ask you to do this as part fo this PR.  I'll add all the relevant tests in #71.

Reply:
Ok, ignored for now.